main.go

```diff
@@ -85,6 +86,16 @@ func main() {
 		log.Println(handlers.DebugEndpointListenAndServe(*debugPort, vodEngine))
 	}()
 
+	// Start cron style apps to run periodically
+	app := "mist-cleanup"
+	// schedule mist-cleanup every 2hrs with a timeout of 5min
```

Comment:
@Thulinma: any concerns/comments with the mist-cleanup script running every 2hrs? What do you suggest?

Reply:
Every two hours is fine - I was personally thinking every hour, but that choice is pretty much arbitrary. As long as we don't run it more than every 15m and not less than once a day, I think we're probably good.